lib/h3core.js

```diff
@@ -420,6 +420,25 @@ function storeCoordIJ(cAddress, {i, j}) {
     C.setValue(cAddress + SZ_INT, j, 'i32');
 }
 
+/**
+ * Read an array of positive integers array from C. Negative
+ * values are considered invalid and ignored in output.
+ * @private
+ * @param  {Number} cAddress    Pointer to C array
+ * @param  {Number} count       Array length
+ * @return {Number[]}           Integer array
```

Comment:
"Javascript integer array" to differentiate from the C array referred to in `cAddress` and `count`?

Reply:
Makes sense, will update.